code/datums/uplink_item.dm

```diff
@@ -960,6 +960,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 1
 	surplus = 20
 
+/datum/uplink_item/stealthy_tools/safecracking
```

Comment:
You exclude the two drills from nuclear emergency but not this?

Reply:
Only one drill is excluded, and only one drill is available in nuclear ops, there's a difference :P

Reply:
Ah my mistake.

Reply:
Also there are times nuke ops will want to stealthily crack the safe. We just made sure they only got the better and cheaper diamond drill, and did not have access to the worse regular version.